<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainCalc</class>
 <widget class="QMainWindow" name="MainCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainCalc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_26">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>sqrt(</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_30">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>DEL</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_12">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_17">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_29">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_25">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButton_22">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="pushButton_20">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_15">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>MC</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_10">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_14">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButton_27">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>RET</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="pushButton_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButton_18">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_13">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QPushButton" name="pushButton_24">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #191970;
	boder: 1px solid gray;
	padding: 5px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QLineEdit" name="lineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	boder: 1px solid gray;
}♣</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_16">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_19">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="pushButton_23">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_11">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>MRC</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_28">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000CD;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
}♣</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #0000FF;
	boder: 1px solid gray;
	padding: 5px;
    width: 56px;
}♣

QPushButton:pressed {
	background-color: #A9A9A9;
	boder: 1px solid gray;
	padding: 5px;
	width: 56px;
}♣
</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
